src/v2/Apps/Conversation/Components/ConversationCTA.tsx

```diff
@@ -27,16 +51,46 @@ export const ConversationCTA: React.FC<ConversationCTAProps> = ({
 
   const conversationID = conversation.internalID!
   const activeOrder = extractNodes(conversation.activeOrders)[0]
+  const {
+    isOfferableFromInquiry,
+    is_offerable: isOfferable,
+    is_acquireable: isAcquireable,
+  } = artwork
 
   if (!activeOrder) {
-    if (artwork.isOfferableFromInquiry) {
-      return (
-        <MakeOfferOnInquiryButtonFragmentContainer
-          openInquiryModal={() => openInquiryModal()}
-          conversation={conversation}
-        />
-      )
-    }
+    return (
+      <>
+        <ShadowSeparator />
+        <Flex flexDirection="column" p={1}>
+          <Flex flexDirection="row">
+            <GuaranteeIconBlue mr={1} />
+            <Flex flexShrink={1}>
+              <Text color="black60" variant="xs" mb={1}>
+                Always complete purchases with our secure checkout in order to
+                be covered by{" "}
+                <RouterLink to="/buyer-guarantee" target="_blank">
+                  The Artsy Guarantee
+                </RouterLink>
+                .
+              </Text>
+            </Flex>
```

Comment:
The Guarantee message was moved here so the only responsibility of `MakeOfferOnInquiryButton` is to be a button.